boa/src/builtins/infinity/mod.rs

```diff
@@ -25,7 +25,7 @@ impl BuiltIn for Infinity {
         Attribute::READONLY | Attribute::NON_ENUMERABLE | Attribute::PERMANENT
     }
 
-    fn init(_context: &mut Context) -> (&'static str, Value, Attribute) {
+    fn init(_: &mut Context) -> (&'static str, Value, Attribute) {
```

Comment:
Why this change?

Reply:
its because the context was not used to changed it to `_` like the other builtin functins